src/Utilities/EmailSender.cs

```diff
@@ -8,6 +8,7 @@ namespace ePiggyWeb.Utilities
 {
     public class EmailSender
     {
+        public const string Email = "Email";
```

Comment:
Where is this used?

Reply:
This is used in the place where you asked your other question, this string is used to access the data from app settings through configuration